### ISSUE TITLE
Aria-expanded attribute is not defined for "More options" button

### ISSUE THREAD

**Issue by melode11** [SeleniumHQ/selenium-ide#1182]
## User Experience
Screen reader user will not able to know the state of the button and can not access the control efficiently if aria-expanded attribute is not defined for the button

## Repro Steps
Open Selenium IDE.
Navigate to  "More options" button and inspect it it
Verify the issue.

## Actual Result
Aria-expanded attribute is not defined for "More options" button due to which screen reader is announcing only "More options" button
## Expected Result
Aria-expanded attribute should be defined for "More options" button

## Note:
Invalid control type is defined for "Complimentary" button appearing beside Add test case button[MAS4.1.2_Invalid control type]

## MAS Reference
MAS 4.1.2 - Name, Role, Value

## Environment

OS: Windows 10
Selenium IDE Version:  3.17.0
Browser: Chrome
Browser Version: 87.0.4280.141

**Comment by melode11** [comment]
PR merged, close issue.
